guides/_posts/2014-01-5-vfl.html

```diff
@@ -259,8 +257,8 @@
 
 {% highlight css %}
 /* VFL */
-@horizontal [#box1]~-~[#box2];
-@vertical [#box3]~-~[#box4];
+@h (#box1)~-~(#box2);
+@v (#box3)~-~(#box4);
```

Comment:
In this case an implicit gap hgap or vgap will be use and bind to the current scope, I agree that hoisting could lead to ambiguity for the user. I'll add a note in the doc.